packages/dds/tree/src/test/domains/json/jsonCursor.spec.ts

```diff
@@ -48,252 +46,6 @@ const testCases: readonly [string, readonly JsonCompatible[]][] = [
     ],
 ];
 
-describe("JsonCursor", () => {
```

Comment:
This set of tests might be handy to point cursor users to as documentation, but again, your call.